apps/grampsweb/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3.7"
+
+services:
+  grampsweb:
+    image: ghcr.io/gramps-project/grampsweb:v24.5.0
+    container_name: grampsweb
+    restart: unless-stopped
+    ports:
+      - ${APP_PORT}:5000 # host:docker
+    environment:
+      GRAMPSWEB_TREE: ${GRAMPS_TREE} # will create a new tree if not exists
```

Comment:
Why do you use a VAR env if not used in the config.json file ? So will be empty forever
Your var is referencing a non-existant var atm.

Reply:
@JigSawFr I think now is correct. Check please